pipeline/transformer/transformer_sierra/src/main/scala/weco/pipeline/transformer/sierra/transformers/SierraTitle.scala

```diff
@@ -23,55 +24,6 @@ object SierraTitle extends SierraDataTransformer with SierraQueryOps {
   //
   // http://www.loc.gov/marc/bibliographic/bd245.html
   def apply(bibData: SierraBibData): Option[String] = {
-    val marc245Field = bibData
-      .nonrepeatableVarfieldWithTag("245")
```

Comment:
This is the only place where this function from SierraQueryOps in scala-libs is being used.  I have inlined it instead.

Reply:
Gotcha, shifting [this](https://github.com/wellcomecollection/scala-libs/blob/6070b4d97408044eac1445836ae2e03e77fd980d/sierra/src/main/scala/weco/sierra/models/SierraQueryOps.scala#L31), over [here](https://github.com/wellcomecollection/catalogue-pipeline/pull/2567/files#diff-896a4df57aa2b93311fde85fa143bed6509fb829f5ed8ca343e736a3981d9ec5R31).